Marlin/Configuration.h

```diff
@@ -2564,23 +2564,28 @@
 //#define DWIN_CREALITY_LCD
 
 //
-// ADS7843/XPT2046 ADC Touchscreen such as ILI9341 2.8
+// Touch Screen Settings(Resistance screen or Capacitance screen)
 //
 //#define TOUCH_SCREEN
 #if ENABLED(TOUCH_SCREEN)
+  #define RES_TOUCH_SCREEN // ADS7843/XPT2046 ADC Touchscreen such as ILI9341 2.8
+  //#define CAP_TOUCH_SCREEN // GT911 Capacitive touch screen such as BIQU_BX_TFT70
+
   #define BUTTON_DELAY_EDIT  50 // (ms) Button repeat delay for edit screens
   #define BUTTON_DELAY_MENU 250 // (ms) Button repeat delay for menus
 
-  #define TOUCH_SCREEN_CALIBRATION
+  #if ENABLED(RES_TOUCH_SCREEN)
```

Comment:
This is not needed, I guess. Because the `Conditionals_LCD.h` can just undef `TOUCH_SCREEN_CALIBRATION` when we have `CAP_TOUCH_SCREEN`